### PR TITLE
Information about addSelect and addMultiselect items as Nette\Utils\Html object

### PR DESCRIPTION
**Pull request by aleswita**
All in commits (so I hope that English its ok), based on this issue: https://github.com/nette/forms/issues/148

### CODE REVIEW

**Comment by dg** [comment]
For disabling should be used `setDisabled(['x', 'y'])`

**Comment by aleswita** [comment]
It's only example, how can use Html object

**Comment by dg** [comment]
But it is incorrect. You should use setDisabled to work properly. Html object should not be used for attributes selected and disabled.

**Comment by aleswita** [comment]
@dg its new for me, look at changes

@dean-huczok tnx!

File: cs/forms.texy [hunk 1]
```diff
@@ -339,6 +339,12 @@ Přidá select box pro výběr více položek (třída [MultiSelectBox |api:Nett
 $form->addMultiSelect('options', 'Možnosti:', $options);
 \--
 
+.[tip]
+Jak u select boxu, tak i u multi select boxu lze jednotlivé prvky nastavit pomocí [Html |api:Nette\Utils\Html] objektu.
+
+.[caution]
```
Comment (f3l1x) [inline cs/forms.texy]: I would remove caution, is it really caution or some suggestion?

File: cs/forms.texy [hunk 2]
```diff
@@ -339,6 +339,12 @@ Přidá select box pro výběr více položek (třída [MultiSelectBox |api:Nett
 $form->addMultiSelect('options', 'Možnosti:', $options);
 \--
 
+.[tip]
+Jak u select boxu, tak i u multi select boxu lze jednotlivé prvky nastavit pomocí [Html |api:Nette\Utils\Html] objektu.
```
Comment (f3l1x) [inline cs/forms.texy]: `..prvky poupravit pomocí..`

File: cs/forms.texy (hunk 1 above)
Comment (f3l1x) [inline cs/forms.texy]: What about simplified solution?

`Vlastnosti jako **selected** a **disabled** se musí nastavit přes formulářové prvky, např. $form->addSelect('foo')->setDisabled().`

File: cs/forms.texy (hunk 2 above)
Comment (dg) [inline cs/forms.texy]: Co třeba `jednotlivé prvky mohou být krom řetězců i objekty Html::('option'), kterým pak lze nastavovat další atributy. Nicméně atributy select a disabled ručně nenastavujte, o ty se stará formulář sám.`

**Comment by f3l1x** [comment]
ping @aleswita

**Comment by f3l1x** [comment]
Thanks @aleswita for first impression, also thanks @dg for update. :+1:

**Pull request closed without merging**